sql/core/src/test/resources/sql-tests/results/udf/postgreSQL/udf-join.sql.out

```diff
@@ -1790,19 +1790,19 @@ SELECT udf(udf('')) AS `xxx`, udf(i), udf(j), udf(t), udf(k)
 -- !query schema
 struct<xxx:string,udf(i):int,udf(j):int,udf(t):string,udf(k):int>
 -- !query output
+	8	8	eight	NULL
```

Comment:
I'm not sure why these changed...

Reply:
can we output `udf(udf(i)), udf(k), udf(t)` in the SELECT list, and check if their values are changed or not?

Reply:
No, the values didn't change, but the order did.

Reply:
This seems wrong given that 8 should definitely not come before 4, and there is an ORDER BY term. I can't quite see how my changes triggered this though.